src/qml/qgismobileapp.qml

```diff
@@ -509,23 +509,49 @@ ApplicationWindow {
     /* Locator Highlight */
     GeometryHighlighter {
       id: locatorHighlightItem
-      //width: 10
-      //color: "yellow"
     }
 
     /* Highlight the currently selected item on the feature list */
     FeatureListSelectionHighlight {
+      id: featureListHighlight
+      visible: !moveFeaturesToolbar.moveFeaturesRequested
+
       selectionModel: featureForm.selection
       mapSettings: mapCanvas.mapSettings
 
-      //model: featureForm.model
-      //selection: featureForm.selection
+      color: "yellow"
+      focusedColor: "#ff7777"
```

Comment:
Aren't these a good candidates for colors going to Theme.qml?

Reply:
I don't think so, I wouldn't be planning to change those colors whenever we implement a dark theme.

If anything, we could eventually look into using the QGIS project's selected color setting which users can modify.